earthmover/operations/row.py

```diff
@@ -116,13 +120,38 @@ def execute(self, data: 'DataFrame', **kwargs):
             """
             super().execute(data, **kwargs)
 
-            if not set(self.columns_list).issubset(data.columns):
+            sort_direc_list = [] # True for ascending
+                          # False for descending
+
+            clean_columns_list = [] 
+                # getting rid of "+" and"-" in front of the column name
+                # when the user inputs the columns in the second format
+
+            for col in self.columns_list:
+
+                if col[0] == "-":
+                    clean_columns_list.append(col.removeprefix("-"))
```

Comment:
Hello @s-yurko , unfortunately `str.removeprefix()` is a 3.9 update, and Earthmover has to maintain backwards compatibility with 3.8. We will have to use a solution that does not use this function.